packages/monorepo-builder/src/FileSystem/ComposerJsonProvider.php

```diff
@@ -64,8 +70,10 @@ public function getRootAndPackageFileInfos(): array
 
     public function getPackageFileInfoByName(string $packageName): SmartFileInfo
     {
-        $packageComposerFiles = $this->packageComposerFinder->getPackageComposerFiles();
-        foreach ($packageComposerFiles as $packageComposerFile) {
+        if (count($this->packageComposerFiles) === 0) {
```

Comment:
```suggestion
        if ($this->packageComposerFiles === []) {
```
There is rector rule for that https://github.com/rectorphp/rector/blob/master/rules/performance/src/Rector/FuncCall/CountArrayToEmptyArrayComparisonRector.php